src/VirtoCommerce.DemoSolutionFeaturesModule.Data/Models/ContactDemoEntity.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using VirtoCommerce.CustomerModule.Core.Model;
+using VirtoCommerce.CustomerModule.Data.Model;
+using VirtoCommerce.DemoSolutionFeaturesModule.Core.Models;
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.DemoSolutionFeaturesModule.Data.Models
+{
+    public class ContactDemoEntity : ContactEntity
+    {
+        [StringLength(128)]
+        public string Title { get; set; }
+
```

Comment:
Remove redundant empty line. Double empty lines are using for separate protected and private methods from public